portfolio/src/main/java/com/google/sps/data/TestCommentObject.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
+
```

Comment:
Even for test objects, it's still good to include JavaDoc explaining what it is/what it's used for/etc.

Reply:
Done!